articles/customization-using-bulma/index.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/customization-using-bulma/
+title: Customize CSS using Bulma
+description: Bulma is a lightweight, open source CSS framework that has excellent customization features, perfect for rapid development.
+author: saiharsha-balasubramaniam
+date: 2020-09-04T00:00:00-08:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/customization-using-bulma/hero.jpg
+    alt: bulma
+---
+
+**Cascading Style Sheets** is a stylesheet language that is used to describe how a webpage looks and feels visually. To speed up the development of projects, developers use CSS frameworks like Bulma. Let us see why CSS frameworks are useful and learn about Bulma.
+
+### Table of Contents
+
+- [Need for a CSS Framework](#need-for-a-css-framework)
+- [Bulma](#bulma)
+- [Getting Started](#getting-started)
+- [Column System](#column-system)
+- [Components](#components)
+- [Customizing Variables](#customizing-variables)
+- [Further Reading](#further-reading)
+
+### Need for a CSS Framework
+
+CSS is an flexible language, with a ton of customization options. This may sometime prove to be overwhelming for a beginner. When you're out to build a product, a CSS framework helps to accelerate the process of development. It helps to reduce time to production and this may prove crucial to a project. Some other benefits to using a CSS framework include:
```

Comment:
Consider revising to "This may sometime**s** be overwhelming" and revise time to production to aid flow.